File: Lab_2/fizzbuzz.py
print("Enter a positie integer between 10 and 100. ")
FizzBuzzCounter = 0
FizzCounter = 0
BuzzCounter = 0
num = 0


while True:
    num = int(input())
    if num>=10 and num<=100:
        break
    print("Invalid input, enter again.")

for x in range(1, num+1):
    if(x%3==0 and x%5==0):
        FizzBuzzCounter +=1
        print("FizzBuzz")
    elif(x%3==0):
        FizzCounter +=1
        print("Fizz")
    elif(x%5==0):
        BuzzCounter +=1
        print("Buzz")
    elif(x%7==0):
        print(x, "is skipped")
    else:
        print(x)

print("--- Summary ---")
print("Fizz count: ", FizzCounter)
print("Buzz count: ", BuzzCounter)
print("FizzBuzz count: ", FizzBuzzCounter)

File: Lab_2/fizzbuzz.cpp
#include <iostream>
using namespace std;

int main() {
    int num = 0;
    int FizzBuzzCounter = 0;
    int FizzCounter = 0;
    int BuzzCounter = 0;

    cout << "Enter a positive integer between 10 and 100: ";

    while (true) {
        cin >> num;

        if (num >= 10 && num <= 100) {
            break;
        }
        cout << "Invalid input, enter again: ";
    }

    for (int x = 1; x <= num; x++) {

        if (x % 3 == 0 && x % 5 == 0) {
            FizzBuzzCounter++;
            cout << "FizzBuzz" << endl;
        }
        else if (x % 3 == 0) {
            FizzCounter++;
            cout << "Fizz" << endl;
        }
        else if (x % 5 == 0) {
            BuzzCounter++;
            cout << "Buzz" << endl;
        }
        else if (x % 7 == 0) {
            cout << x << " is skipped" << endl;
        }
        else {
            cout << x << endl;
        }
    }

    cout << "--- Summary ---" << endl;
    cout << "Fizz count: " << FizzCounter << endl;
    cout << "Buzz count: " << BuzzCounter << endl;
    cout << "FizzBuzz count: " << FizzBuzzCounter << endl;

    return 0;
}